### PR TITLE
Serverless containers

### PR DESCRIPTION
**Pull request by tomkins**
This is used to generate https://github.com/developersociety/sdafmhserverless and https://github.com/developersociety/flippybot

To test:

```
mktmpenv
cookiecutter gh:developersociety/serverless-template -c serverless-containers
cd projectname
nvm use
make reset
```

There's possibly more minor tweaks at some point - however those can always be later PRs!

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+on: pull_request
+jobs:
+  test:
+    name: Test -- ${{ matrix.testenv }}
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        testenv:
+          - serverless
```
Comment (tomkins) [inline .github/workflows/ci.yml:11]: Not much in the matrix, however it might be nice to have other options ready at some point (eg. one without Flask!).

**Comment by ikonitas** [comment]
Have you got an antidote for this?

```
└─[15:20]$ cookiecutter gh:developersociety/serverless-template -c serverless-containers
/usr/local/Cellar/pyenv/1.2.26_1/libexec/pyenv-exec: /Users/ikonitas/.local/bin/cookiecutter: /Users/ikonitas/.pyenv/versions/3.8.0/bin/python3.8: bad interpreter: No such file or directory
/usr/local/Cellar/pyenv/1.2.26_1/libexec/pyenv-exec: line 48: /Users/ikonitas/.local/bin/cookiecutter: Undefined error: 0
```

**Comment by tomkins** [comment]
Couple of things:

`pyenv rehash` - though possibly not the fix.

Running pipx? Might need to reinstall cookiecutter.

**Review by ikonitas — APPROVED** [review]
I have updated the cookie-cutter and it's working!  Looking at the pipx

**Pull request merged**